templates/observability/index.html

```diff
@@ -634,7 +634,11 @@ <h3>Open source, simplified</h3>
   <hr />
 </div>
 
-<section class="p-strip is-deep">
+{% with section_classes='p-strip', heading_topic='Observability', tag_name='observability', tag_id='4125', limit='4' %}
+  {% include "shared/_latest_news_strip.html" %}
+{% endwith %}
+
+<section class="p-strip">
```

Comment:
I'm no designer but this might fit better if it was `p-strip--light`

Reply:
Yeah, I tried it locally and I like it better :-)